### ISSUE TITLE
How we can clear selected date from calendar 

### ISSUE THREAD

**Issue by jawadg** [RafalManka/ScrollCalendar#8]

**Comment by RafalManka** [comment]
Return something other than "CalendarDay.SELECTED" e.g.

```
scrollCalendar.setDateWatcher(new DateWatcher() {
    @Override
    public int getStateForDate(int year, int month, int day) {
        return CalendarDay.DEFAULT;
    }
});
```

**Comment by osoffer** [comment]
Is there a way, after ScrollCalendar was inflated, to change the selected dates by code (and not by user clicks)? 
something like a "refreshDates()".

I'm asking for the purpose of giving the user the option to cancel his current date selection and restore the previous selection the next time the user opens the view.

Right now I'm doing so by removing the exsiiting ScrollCalendar view and creating a new one (a fairly costly action, instead of making it invisible and refreshing the dates).

TNX.

**Comment by RafalManka** [comment]
I just released a new version of ScrollCalendar (1.5.2) that exposes a refresh() method for you. now when you want to change the state of the widget programmatically e.g. when the user clicks a "CLEAR" button, call  this method:
```
scrollCalendar.refresh()
```

Check it out and tell me if this is what you've been looking for.

**Comment by osoffer** [comment]
Tried it, works perfectly.

TNX!
